packages/fluentui/react-northstar/src/themes/teams/components/Breadcrumb/breadcrumbLinkStyles.ts

```diff
@@ -0,0 +1,70 @@
+import { BreadcrumbLinkStylesProps } from '../../../../components/Breadcrumb/BreadcrumbLink';
+import { ComponentSlotStylesPrepared, ICSSInJSStyle } from '@fluentui/styles';
+import { getBorderFocusStyles } from '../../getBorderFocusStyles';
+import { BreadcrumbVariables } from './breadcrumbVariables';
+
+export const breadcrumbLinkStyles: ComponentSlotStylesPrepared<BreadcrumbLinkStylesProps, BreadcrumbVariables> = {
+  root: ({ props: p, variables: v, theme: { siteVariables } }): ICSSInJSStyle => {
+    const borderFocusStyles = getBorderFocusStyles({
+      variables: siteVariables,
+      ...(p.size === 'smaller' && {
+        borderPadding: {
+          top: v.linkPaddingTop,
+          right: v.linkPaddingRightSmaller,
+          bottom: v.linkPaddingBottom,
+          left: v.linkPaddingLeftSmaller,
+        },
+      }),
+      ...(p.size === 'small' && {
+        borderPadding: {
+          top: v.linkPaddingTop,
+          right: v.linkPaddingRightSmall,
+          bottom: v.linkPaddingBottom,
+          left: v.linkPaddingLeftSmall,
+        },
+      }),
+      ...(p.size === 'medium' && {
+        borderPadding: {
+          top: v.linkPaddingTop,
+          right: v.linkPaddingRightMedium,
+          bottom: v.linkPaddingBottom,
+          left: v.linkPaddingLeftMedium,
+        },
+      }),
+      ...(p.size === 'large' && {
+        borderPadding: {
+          top: v.linkPaddingTop,
+          right: v.linkPaddingRightLarge,
+          bottom: v.linkPaddingBottom,
+          left: v.linkPaddingLeftLarge,
+        },
+      }),
+    });
+
+    return {
+      display: 'flex',
+      alignItems: 'center',
+      position: 'relative',
+      ':visited': {
+        color: 'inherit',
+      },
+      ':focus': borderFocusStyles[':focus'],
+      ':focus-visible': {
+        ...borderFocusStyles[':focus-visible'],
+      },
+
+      ...(p.size === 'smaller' && {
+        gap: v.linkSmallerGap,
```

Comment:
Our customer moves to Electron 8, it runs on Chromium 80, I would not be so optimistic about `gap` with Flexbox:

![image](https://user-images.githubusercontent.com/14183168/94263851-fc550000-ff35-11ea-8e66-4c91090e7fc0.png)

Reply:
removed gap